ci/scripts/connect-release-init.js

```diff
@@ -179,7 +177,13 @@ const initConnectRelease = () => {
     // Creating a comment only if there are commits to add since last connect release.
     if (connectGitLogIndex !== -1) {
         connectGitLogArr.splice(connectGitLogIndex, connectGitLogArr.length - connectGitLogIndex);
-        const connectGitLogText = formArrayToText(connectGitLogArr);
+        // In array `connectGitLogArr` each item string contains " at the beginning and " at the end, let's remove those characters.
```

Comment:
interesting. where these characters come from in the first place? 🤔

Reply:
I comes from "getGitCommitByPackageName".
We also removed it in the other place where we use it in the same script but like: 
```
newCommits.push(commit.replaceAll('"', ''));
```
I just realized we use there as well.